Assets/SteamVR_Unity_Toolkit/Scripts/VRTK_InteractableObject.cs

```diff
@@ -283,13 +294,20 @@ public void SaveCurrentState()
             if (grabbingObject == null)
             {
                 previousParent = transform.parent;
-                previousKinematicState = rb.isKinematic;
+
+                if (rb)
```

Comment:
was this causing you issues? I think a interactable object should always have a rigidbody?